### ISSUE TITLE
Adding duplicate resources

### ISSUE THREAD

**Issue by clash99** [Cadasta/cadasta-platform#296]
It would be great if adding a duplicate resource (by name comparison perhaps?), it would alert you if that resource already existed and ask you if you wanted to connect to it, or continue to upload your new resource (which I would assume at this point is not a duplicate).

**Comment by wonderchook** [comment]
What about comparing a hash of the file to what is already uploaded in the project? @oliverroick did we talk about this at some point or is my memory bad?

**Comment by oliverroick** [comment]
We did talk about updating XLSForms. 

Using hashes to look for existing files is something we can definitely do. Do bear in mind, though: We went a long way to making sure files don't have to go through our servers when uploading resources. Instead, upload them to S3 directly and just store its URL. If we want to check if a file exists, we have to download and process the files every time a resource is being created.
